src/components/victory-axis/tick-label.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from "react";
+import { VictoryLabel, Helpers } from "victory-core";
+
+export default class TickLabel extends React.Component {
+  static role = "tick";
+
+  static propTypes = {
+    position: PropTypes.object,
+    tick: PropTypes.any,
+    orientation: PropTypes.oneOf(["top", "bottom", "left", "right"]),
+    isVertical: PropTypes.bool,
+    style: PropTypes.object,
+    events: PropTypes.object,
+    label: PropTypes.any
+  };
+
+  getAnchors(props) {
+    const anchorOrientation = { top: "end", left: "end", right: "start", bottom: "start" };
+    const anchor = anchorOrientation[props.orientation];
+    return {
+      textAnchor: props.isVertical ? anchor : "middle",
+      verticalAnchor: props.isVertical ? "middle" : anchor
+    };
+  }
+
+  render() {
+    if (!this.props.label) {
+      return undefined;
+    }
+    const componentProps = this.props.label.props ? this.props.label.props : {};
```

Comment:
Nit: Can be simplified to `const componentProps = this.props.label.props || {}; `